### PR TITLE
Add Tumbleweed support for LUKS unlock via SSH (via dracut-ssh)

### PR DESCRIPTION
**Pull request by STARRY-S**
1. Added YAML schedule and autoyast XML files for TW.
2. Updated the test code for TW.

- Related ticket: https://progress.opensuse.org/issues/110953
- Needles: 
    There are two needles created:
    - https://openqa.opensuse.org/tests/2345181#step/shutdown/5
    - https://openqa.opensuse.org/tests/2345245#step/unlock_via_ssh_client/11
- Verification run: 
    Tumbleweed:
    - https://openqa.opensuse.org/tests/2349994
    - https://openqa.opensuse.org/tests/2350014
    - https://openqa.opensuse.org/tests/2350234
    sle15sp4:
    - http://openqa.suse.de/tests/8750680
    - http://openqa.suse.de/tests/8750681

### CODE REVIEW

File: tests/security/luks1/unlock_via_ssh_server.pm [hunk 1]
```diff
@@ -45,7 +45,7 @@ sub run {
     wait_for_children;
 
     # Make sure the boot is unlocked and server should be up
-    assert_screen('displaymanager', 200);
+    assert_screen([qw(generic-desktop opensuse-welcome displaymanager)]);
```
Comment (rfan1) [inline tests/security/luks1/unlock_via_ssh_server.pm:48]: pls add timeout=200 back,
btw, did you add new needle here? I don't think so  :)

File: tests/security/luks1/unlock_via_ssh_client.pm [hunk 2]
```diff
@@ -33,20 +33,23 @@ sub run {
...
+    wait_still_screen(5);
     save_screenshot;
     enter_cmd("$testapi::password");
-    sleep 5;
-    enter_cmd('exit');
-    sleep 5;
+    wait_still_screen(5);
+    save_screenshot;
+    # exit from ssh session if it is not closed by remote host
+    enter_cmd('exit') if !check_screen('ssh_connection_closed_by_remote_host');
```
Comment (rfan1) [inline tests/security/luks1/unlock_via_ssh_client.pm:51]: how about "unless"
anyway, not a big deal

File: tests/security/luks1/unlock_via_ssh_server.pm (hunk 1 above)
Comment (STARRY-S) [reply tests/security/luks1/unlock_via_ssh_server.pm]: Thanks for the comments,
I did not create new needle here.

File: tests/security/luks1/unlock_via_ssh_client.pm (hunk 2 above)
Comment (STARRY-S) [reply tests/security/luks1/unlock_via_ssh_client.pm]: Done, thanks.

**Review by Amrysliu — APPROVED** [review]
LGTM

**Review by rfan1 — APPROVED** [review]

**Review by lilyeyes — APPROVED** [review]
LGTM

**Pull request merged**